static/js/reducers/financial_aid_test.js

```diff
@@ -122,4 +127,28 @@ describe('financial aid reducers', () => {
       assert.deepEqual(state, expectation);
     });
   });
+
+  it('should let you skip financial aid', () => {
+    skipFinancialAidStub.returns(Promise.resolve());
+    return dispatchThen(skipFinancialAid(2), [
+      REQUEST_SKIP_FINANCIAL_AID,
+      RECEIVE_SKIP_FINANCIAL_AID_SUCCESS
+    ]).then(state => {
+      assert.deepEqual(state, {
```

Comment:
Can you assert what arguments `skipFinancialAidStub` was called with, here and below?